definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -957,5 +957,89 @@
     "CaseFieldID": "FurtherHWFReferenceNumbers",
     "UserRole": "citizen",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "11/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LastServiceApplicationGranted",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "11/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LastServiceApplicationGranted",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "11/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LastServiceApplicationGranted",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "11/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LastServiceApplicationGranted",
+    "UserRole": "caseworker-divorce-courtadmin",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "11/01/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "LastServiceApplicationGranted",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "CRUD"
```

Comment:
Are you sure you need these three roles to have CRUD rights to this field?

Reply:
I am not entirely sure but I thought it would be safer to copy the permissions of the existing fields because I remember that we had added the "CRUD" roles because of a bug we had noticed. Just to avoid having anything like this with those fields I thought it it just easier to give the same permissions.
Also, having 1 complex type field could work assuming that showConditions work well with Complex type, so that we can do: ServiceApplication.type = "true" for example.

Reply:
If we can make sure we only have the necessary CRUD rights, that'd be better. I'm especially worried about the solicitor having it, if not needed.
As to the complex type, I think sub-field reference works (although I'm not sure).
Happy for you to merge this and create a sub-task on your story to go through these comments (if you think they make sense).